server/src/test/java/org/candlepin/dto/api/v1/ComplianceReasonTranslatorTest.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2009 - 2017 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import static org.junit.Assert.*;
+
+import org.candlepin.dto.AbstractTranslatorTest;
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.policy.js.compliance.ComplianceReason;
+
+import org.junit.runner.RunWith;
+
+import junitparams.JUnitParamsRunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+
+/**
+ * Test suite for the ComplianceReasonTranslator class
+ */
+@RunWith(JUnitParamsRunner.class)
+public class ComplianceReasonTranslatorTest extends
+    AbstractTranslatorTest<ComplianceReason, ComplianceReasonDTO, ComplianceReasonTranslator> {
+
+    protected ComplianceReasonTranslator ComplianceReasonTranslator = new ComplianceReasonTranslator();
+
+    @Override
+    protected void initModelTranslator(ModelTranslator translator) {
+        translator.registerTranslator(this.ComplianceReasonTranslator, ComplianceReason.class,
+            ComplianceReasonDTO.class);
+    }
+
+    @Override
+    protected ComplianceReasonTranslator initObjectTranslator() {
+        return this.ComplianceReasonTranslator;
```

Comment:
I know I'm being a bit petty, but reading this and taking a while to realize its a variable starting with capital letter triggered me a bit :D

Reply:
Refactoring artifact, you're right to call these out. Fixed.